assembl/static2/js/app/utils/globalFunctions.js

```diff
@@ -17,6 +17,11 @@ export const getDiscussionId = () => getInputValue('discussion-id');
 
 export const getDiscussionSlug = () => getInputValue('discussion-slug');
 
+export const formatNumber = (number: number | string, locale: string) => {
+  const separator = locale !== 'en' ? ' ' : ',';
+  return number.toString().replace(/\B(?=(\d{3})+(?!\d))/g, separator);
+};
+
```

Comment:
You are only supporting two languages here. Please use existing lib for this. Look at react-redux-i18n Localize component for this which uses the standard Intl.NumberFormat behind the scene.

Reply:
Well the story only talks about dealing with english and every other language should have whitespaces. But I agree if we have a simple way to deal with every language why not. Need to take a look at this, I didn't know about it